wifiphisher/pywifiphisher.py

```diff
@@ -404,6 +409,11 @@ def start(self):
         if args.credential_log_path:
             phishinghttp.credential_log_path = args.credential_log_path
 
+        # Handle the chosen interface as an internetInterface in order to 
+        # leverage existing functionality. 
+        # In case `--internetinterface` is also used it will be overwritten silently.
+        if args.mitminterface:
+            args.internetinterface = args.mitminterface
```

Comment:
Since you are enabling `args.internetinterface` here, this will *always* result in the new warning: `using  both --mitminterface (-mI) and --internetinterface (-iI) is redundant. Ignoring --internetinterface (-iI).`.

Reply:
It did not occur to me while testing but It could be prevented indeed.  
 
I made some changes to overwrite the value of `args.mitminterface` to a specific string in order to be able to use its presence for further checks in later steps. Since we will be using the `args.internetinterfaces` object for the various operations, overwriting `args.mitminterface` should have no real effect. What do you think?

Reply:
Well, my logic was flawed, but I did fix that eventually in the latest commit.